src/plugin.js

```diff
@@ -83,94 +83,68 @@
     });
   });
 
+  // We want 4.x-style sizing; so, new methods that are related, but are
+  // not used by 4.x-style resizing are rendered no-op.
+  ['aspectRatio', 'fluid', 'updateStyleEl_'].forEach(function(methodName) {
+    originals.Player[methodName] = Player.prototype[methodName];
+    Player.prototype[methodName] = Function.prototype;
+  });
 
-
-  // Backfill support for %-based dimensions in players, which was
-  // supported in video.js 4.x.
-  Player.prototype.dimension = function(dimension, value) {
-
-    // When the value is set as a %, bypass the usual handling.
-    if (isPct(value)) {
-      this[dimension + '_'] = value;
-      this.updateStyleEl_();
-      return this;
+  // We want to un-create the styleEl_ because for 4.x-style dimensions, we
+  // don't need the style element confusing the issue.
+  Player.prototype.createEl = function() {
+    var el = originals.Player.createEl.apply(this, arguments);
+    if (this.styleEl_) {
+      this.styleEl_.parentNode.removeChild(this.styleEl_);
     }
-
-    // In all other cases, use the video.js 5.x handling.
-    return originals.Player.dimension.call(this, dimension, value);
+    return el;
   };
 
-  // This method must be overridden because if the original video.js 5.x
-  // method is called on a player with percentage sizing, the dimension
-  // in CSS will be set to "100%px" as an example. Also, this sets styles
-  // inline instead of writing to a stylesheet because that's how 4.x
-  // worked! Most of this method is copy/pasted... unfortunately.
-  Player.prototype.updateStyleEl_ = function() {
-    var aspectRatio, ratioParts, ratioMultiplier, width, height, idClass, styles;
-
-    // The aspect ratio is either used directly or to calculate width and height.
-    if (this.aspectRatio_ !== undefined && this.aspectRatio_ !== 'auto') {
-      // Use any aspectRatio that's been specifically set
-      aspectRatio = this.aspectRatio_;
-    } else if (this.videoWidth()) {
-      // Otherwise try to get the aspect ratio from the video metadata
-      aspectRatio = this.videoWidth() + ':' + this.videoHeight();
-    } else {
-      // Or use a default. The video element's is 2:1, but 16:9 is more common.
-      aspectRatio = '16:9';
-    }
+  // Roll back player dimension behavior to 4.x compatibility.
+  Player.prototype.dimension = function(dimension, num, skipListeners) {
```

Comment:
Yes! Good call. I didn't realize that method was the same (and didn't think to check). Will simplify this...